packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.15.0`
+
+- Added the secure credentials support for Extenders API [#1306](https://github.com/zowe/vscode-extension-for-zowe/pull/1306). Thanks @JillieBeanSim
+- Improved the Command Palette by adding 'Zowe Explorer:` before all commands that are related to the extension. Removed some commands from the palette that caused issues [#1308](https://github.com/zowe/vscode-extension-for-zowe/pull/1308). Thanks @lauren-li
```

Comment:
Looks like the backtick for the `Zowe Explorer:` tag should be fixed

Reply:
yes :D I couldn't figure this out last night!